.github/workflows/main.yml

```
@@ -1,6 +1,7 @@
 on:
   schedule:
    - cron: '0 2 * * *'
+  push:
 
 jobs:
   delete_old_archives:
```

This is a copy/paste error from when I made the repo, might as well change it now...
```suggestion
  update_gitstats:
```

---

Ah, I just changed the job name to `mathlib_stats` before noticing your suggestion.